BonusCalcApi/V1/Controllers/WorkElementsController.cs

```diff
@@ -1,3 +1,4 @@
+using Asp.Versioning;
```

Comment:
If this is actually necessary, I'm amazed how at how the build didn't fail without it.

Reply:
Yes.